src/fetch.js

```diff
@@ -2,28 +2,28 @@ import fetchSetter from './index.js';
 
 let fetcher = {
  fetchUser(id) {
-    fetch(`https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers/${id}`)
-        .then(response => response.json())
-        .then(data => fetchSetter.setUserData(data))
-        .then(this.fetchTripsForAUser())
-        .catch(err => console.log('err', err))
+     const fetchedUser = `https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers/${id}`
+     const promise =  fetch(fetchedUser)
+     .then(response => response.json())
+     return promise
+     .catch(err => console.log('err', err))
```

Comment:
Wow? Really? Twice?....I trusted you......

Reply:
https://media.giphy.com/media/7ZvPlxBHwfK1y/giphy.gif